garnett/trajectory.py

```diff
@@ -494,66 +502,102 @@ def types(self, value):
         self.frame_data.types = value
 
     @property
-    def positions(self):
+    def position(self):
         "Nx3 array of coordinates for N particles in 3 dimensions."
         self.load()
-        return self._raise_attributeerror('positions')
+        return self._raise_attributeerror('position')
 
-    @positions.setter
-    def positions(self, value):
+    @property
+    def positions(self):
+        """
+        Nx3 array of coordinated for N particles in 3 dimensions.
+        Deprecated alias for position.
+        """
+        self._deprecation_warning('positions', 'position')
+        try:
+            return self.position(self)
```

Comment:
Why is this line necessary? I think it should just return the property, `return self.position`?